client/account-settings/index.js

```diff
@@ -2,12 +2,12 @@
  * External dependencies
  */
 import React from 'react';
-import { combineReducers } from 'redux';
+import { combineReducers } from 'state/utils';
```

Comment:
We don't have a `combineReducers` in `state/utils` - did you mean to import it from Calypso (it's what they use)? I don't think we need to import from Calypso since we're not using any of the state persistence logic, which is what that function was created for.